packages/amplify-graphql-model-transformer/src/resources/amplify-dynamodb-table/amplify-dynamo-model-resource-generator.ts

```diff
@@ -5,22 +5,19 @@ import { ObjectTypeDefinitionNode } from 'graphql';
 import { setResourceName } from '@aws-amplify/graphql-transformer-core';
 import { Table } from 'aws-cdk-lib/aws-dynamodb';
 import { Construct } from 'constructs';
-import { DynamoDBModelVTLGenerator, ModelVTLGenerator } from '../resolvers';
 
 import { Duration, aws_iam, aws_lambda, custom_resources, aws_logs } from 'aws-cdk-lib';
 import { CustomResource } from 'aws-cdk-lib';
-import { DynamoModelResourceGenerator } from './dynamo-model-resource-generator';
+import { DynamoModelResourceGenerator } from '../dynamo-model-resource-generator';
 import * as path from 'path';
 
 export const ITERATIVE_TABLE_STACK_NAME = 'AmplifyTableManager';
-export const CUSTOM_DDB_CFN_TYPE = 'Custom::AmplifyManagedDynamoDBTable';
+export const CUSTOM_DDB_CFN_TYPE = 'Custom::AmplifyDynamoDBTable';
 /**
- * DynamoModelResourceGenerator is an implementation of ModelResourceGenerator,
- * providing necessary utilities to generate the DynamoDB resources for models
+ * AmplifyDynamoModelResourceGenerator is an subclass of DynamoModelResourceGenerator,
```

Comment:
nit to incorporate into your next PR (no need to go through the entire review/approval/testing process again on this PR)
```suggestion
 * AmplifyDynamoModelResourceGenerator is a subclass of DynamoModelResourceGenerator,
```